### DISCUSSION TITLE
Postal 2.1.4: 403 Forbidden

### DISCUSSION THREAD

**Discussion by dietmarscharf** [postalserver/postal#2437] [Help with using Postal]
In case you receive an 403 http error "Forbidden" (or an empty page in some browsers) when accessing the postal web server locally on http://127.0.0.1:5000, switch to the hostname specified for postal web in the postal.yam config file, so http://<hostname>:5000 and add this hostname to your /etc/hosts file to resolve it to 127.0.0.1. This problem / strict host name check didn't happen with 2.1.3 or 2.1.2 for my installations. Hope this report helps someone experiencing the same issue!

**Comment by willpower232** [comment]
Ideally you would use a proxy such as nginx or caddy with a real domain name rather than access the port directly. This also means you can apply HTTPS.

**Comment by dietmarscharf** [comment]
I use a Cloudflare tunnel instead of caddy or nginx, provides WAF and endpoint security as easy as it could be: https://hub.docker.com/r/cloudflare/cloudflared

**Reply by uhlhosting** [reply]
@dietmarscharf you can change that for yourself, I one would never use anything cloudflare, and so many others I bet they are in same position. 

This is not a suggestion area, it seems you started by presenting a issue, then you ended up making suggestions, first ask yourself what Open Source is, and how it works. 

Clearly you missed maybe that Postal is open source. So feel free to fork it, and add your cloudflare proxy in it. No need to make such suggestions here, they are in vain. 

A correct path would be, if the community really have a need for such in first place, and we clearly do not, or we would had long had it.

- Fork the project
- Make changes/improvements
- Test it yourself first
- Push a PR
